eng/Versions.props

```diff
@@ -205,7 +206,7 @@
          Additionally, set the MinimumVSVersion for the installer UI that's required for targeting NetCurrent -->
     <MicrosoftBuildVersion>17.15.0-preview-25230-108</MicrosoftBuildVersion>
     <MicrosoftBuildLocalizationVersion>17.15.0-preview-25230-108</MicrosoftBuildLocalizationVersion>
-    <MicrosoftBuildMinimumVersion Condition="'$(DotNetBuildSourceOnly)' != 'true'">17.11.4</MicrosoftBuildMinimumVersion>
+    <MicrosoftBuildMinimumVersion Condition="'$(DotNetBuildSourceOnly)' != 'true'">17.13.9</MicrosoftBuildMinimumVersion>
     <MinimumVSVersion>17.13</MinimumVSVersion>
```

Comment:
Should we consider adding a check somewhere to compare `MicrosoftBuildMinimumVersion` (major.minor) to the `MinimumVSVersion` value below and trigger an error if they do not line up?

Reply:
I'm trying to think if there's ever a scenario where we would want them to misalign. This is the version our tasks build against and the other min version is the lowest we load in. So probably always the same major/minor?

Reply:
yeah, that's what I'm thinking, unless there can be a misalignment when we jump VS versions, but not msbuild, e.g. if agents are updated and we're missing flows, though we don't compare these with the installed VS version and the two value should come from us. It was just a thought - I have no idea where we'd hook up the validation. Probably some place weird like the directory.build.targets in the root

Reply:
I added a check in redist.csproj if you want to take a look. Seems to work from testing. I don't know if there's a different way to do this.